entrypoint.sh

```diff
@@ -15,53 +15,69 @@ SCRIPT_AFTER_MAKE_TEST="`pwd`/.github/ci-bionic/after_make_test.sh"
 
 cd $GITHUB_WORKSPACE
 
-echo ::group::Dependencies from binaries
+echo ::group::Install tools: apt
 apt update
 apt -y install wget lsb-release gnupg
 sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" > /etc/apt/sources.list.d/gazebo-stable.list'
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
-apt-get update
+apt-get update 2>&1
 apt -y install \
   cmake \
   build-essential \
   curl \
   g++-8 \
   git \
   cppcheck \
-  python3-pip \
-  $APT_DEPENDENCIES
+  python3-pip
+echo ::endgroup::
 
+echo ::group::Install tools: pip
 pip3 install -U pip vcstool colcon-common-extensions
 echo ::endgroup::
 
+echo ::group::Install tools: source
+git clone https://github.com/linux-test-project/lcov.git -b v1.14 2>&1
+cd lcov
+make install
+cd ..
+echo ::endgroup::
+
 echo ::group::GCC 8
 update-alternatives \
   --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 \
   --slave /usr/bin/g++ g++ /usr/bin/g++-8 \
   --slave /usr/bin/gcov gcov /usr/bin/gcov-8
 echo ::endgroup::
 
-echo ::group::Install LCOV
-git clone https://github.com/linux-test-project/lcov.git -b v1.14 2>&1
-cd lcov
-make install
-cd ..
+echo ::group::Fetch source dependencies
+if [ -f "$SOURCE_DEPENDENCIES" ] ; then
+  mkdir -p deps/src
+  cd deps
```

Comment:
I think we can condense the three lines in: `vcs import deps/src ...` and get rid of the cd commands.

Reply:
Done 0c36316